### PR TITLE
feature/changes to publish the library on npm

### PR DESCRIPTION
**Pull request by alexanderzulechner**
- ScrollingFix: Changing scroll detection algorithm to use relative scroll distances
- ScrollingFix: Preventing negative offset values
- ScrollingFix: Changing start/end to be the rendered start and end
- StickyFix: Last test fixes
- fix: fixes behaviour where upwards scrolling of first rendered data did cut off half of the header, if header and rows are not equal in height
- chore: changes everything to publish to npm

### CODE REVIEW

**Comment by philippspindler** [comment]
Looks good 👍 

Not sure if you are allowed to remove the author with MIT License 🤔

File: package.json [hunk 1]
```diff
@@ -1,18 +1,11 @@
 {
-  "name": "ng-table-virtual-scroll",
-  "version": "1.3.6",
+  "name": "@craftworks/ng-table-virtual-scroll",
+  "version": "1.3.7",
   "description": "Virtual scroll for for Angular Material Table",
```
Comment (lukashaselsteiner) [inline package.json]: maybe we can add the link to the original homepage here in the description?

```suggestion
  "description": "Virtual scroll for for Angular Material Table, fork of https://github.com/diprokon/ng-table-virtual-scroll",
```

File: README.md [hunk 2]
```diff
@@ -1,106 +1,3 @@
 # Virtual Scroll for Angular Material Table
 
-An Angular Directive, which allow to use [virtual scrolling](https://material.angular.io/cdk/scrolling) in [mat-table](https://material.angular.io/components/table)
```
Comment (lukashaselsteiner) [inline README.md:3]: i wouldn't remove the whole readme, but just add the 'Origin Library was forked...' as first sentence here? and keep the rest of the original readme?

File: package.json (hunk 1 above)
Comment (alexanderzulechner) [reply package.json]: like how else? this the link to the original repo?

Comment (lukashaselsteiner) [reply package.json]: isn't this the original repo link. i just thought maybe its a good idea to add this link to the description. but not sure :)

**Comment by alexanderzulechner** [comment]
> Looks good 👍
> 
> Not sure if you are allowed to remove the author with MIT License 🤔

@philippspindler but I didn't remove the license file and the origin author from it?

**Review by boianvelitchkov — APPROVED** [review]
very good job 🚀

**Comment by philippspindler** [comment]
> @philippspindler but I didn't remove the license file and the origin author from it?

looks good, how you did it now

**Review by jakoblahmer — APPROVED** [review]
👍  looks good, nice work

**Comment by alexanderzulechner** [comment]
> > @philippspindler but I didn't remove the license file and the origin author from it?
> 
> looks good, how you did it now

ah u meant the info in the package.json ? that's the only thing I was not sure about.. because actually, if we publish it, aren't we the author?

the license file with the copyright of the creator did always stay in this PR

**Pull request merged**